content/zh/docs/setup/install/helm/index.md

```diff
@@ -8,8 +8,7 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
-请跟随本指南一起，使用
- [Helm](https://helm.sh/docs/) 安装、配置、并深入评估 Istio 网格系统。
+请跟随本指南一起，使用 [Helm](https://helm.sh/docs/) 安装、配置、并深入评估 Istio 网格系统。
```

Comment:
```suggestion
请遵循本指南的说明，使用 [Helm](https://helm.sh/docs/) 安装和配置 Istio 网格系统。
```
The en text is inconsistent with zh currently. Hope you can check this page through.